client/signup/signup-header/index.jsx

```diff
@@ -19,11 +20,7 @@ export default class SignupHeader extends Component {
 		return (
 			<div className="signup-header">
 				<WordPressLogo size={ 120 } className={ logoClasses } />
-
-				{ /* Ideally, this is where the back button
-			   would live. But thats hard to move, it seems. */ }
-				<div className="signup-header__left" />
```

Comment:
Neat. Please also delete the [CSS for this](https://github.com/Automattic/wp-calypso/blob/add/newsletter-start-flow/client/signup/signup-header/style.scss#L45-L49).

Reply:
👍